cli_test2.py

```diff
@@ -0,0 +1,49 @@
+'''
+Testing file (TY)
+ - I likely will be writing my own test files
+    since my TextEdit crashes sometimes when loading your code... orz
+'''
+
+#==================================================#
+'''Import'''
+
+import toml
+from logic.common import file_utils as fu
```

Comment:
Alright I'll keep it long then.

Still, I prefer referring the modules in different names since it means I won't have to scan through the whole file to change the reference if we end up renaming the local files. You _can_ techncially use the "Search & Replace" functionality of most test-editing apps, but that might be error-prone when we have functions named similarly.

Anyway, what's your preferenece on this?

Reply:
The situation you're referring to is not a problem. For example, say we start with:

`from logic.common import file_utils`

if in the future, we changed file_utils.py to be utils_file, we'd have to edit the top import line to be one of the following:

1. `from logic.common import utils_file`
2. `from logic.common import utils_file as file_utils`

If we go with 2, then we don't have to change the usages of file_utils through the entirety of the file. But the point remains, that we'd have to edit the same line anyway. Therefore, we can make the determination to go with 1 or 2 when the time actually comes.